UI/styles.css

```diff
@@ -0,0 +1,190 @@
+body {
+  background: radial-gradient(#F5F5DC, #00BFFF30);
+}
+
+.header {
+  font-size: 30px;
+  border-radius: 5px;
+  background: linear-gradient(to left, #00BFFFaa, #00BFFF2a);
+  height:70px;
+  width:100%;
+}
+
+.Main {
+  width:50px;
+  height:30px;
+  font-size:16px;
+  margin-top:15px;
+  margin-left:20%;
+  border:0px;
+  background-color: #00000020;
+  color:#303030;
+}
+
+.search_post {
+  position: relative;
+  width:60px;
+  height:40px;
+  top:-10px;
+  margin-left:7%;
+  background-color: #00000020;
+  color:#303030;
+  font-size: 16px;
+  border:0px;
+}
+
+.add_post {
+  position: relative;
+  margin-left:3%;
+  top:-10px;
+  width:60px;
+  height:40px;
+  border:0px;
+  font-size: 16px;
+  background-color: #00000020;
+  color:#303030;
+}
+
+.profile_photo {
+  position: relative;
+  top:15px;
+  margin-left:28%;
+  width:60px;
+  height:40px;
+}
+
+.profile_name {
+  margin-left:2%;
+  font-size: 20px;
+  color:#000000;
+}
+
+.logout {
+  width:70px;
+  height:30px;
+  margin-left:2%;
+  background-color: #00000020;
+  color:#303030;
+  font-size: 16px;
+  border:0px;
+}
+
+.post {
+  margin:20px 30% 70px 30%;
+  font-size:20px;
+  border:0px;
+}
+
+.header_post {
+  background: #50C8789a;
+  border-top-left-radius: 15px;
+  border-top-right-radius: 15px;
+  border:1px solid black;
+  height:70px;
+  width:100%;
+  font-size: 20px;
+}
+
+.avatar {
+  margin-top:10px;
+  margin-left:10px;
+  width:50px;
+  height:50px;
+}
+
+.username {
+  margin-top:-45px;
+  margin-left:70px;
+  font-size: 24px;
+  color:#303030;
+}
+
+.date {
```

Comment:
It's a bad idea to align items via margins, furthermore, negative. It can be done using [flex element](https://developer.mozilla.org/ru/docs/Web/CSS/CSS_Flexible_Box_Layout/%D0%92%D1%8B%D1%80%D0%B0%D0%B2%D0%BD%D0%B8%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5_%D1%8D%D0%BB%D0%B5%D0%BC%D0%B5%D0%BD%D1%82%D0%BE%D0%B2_%D0%B2_Flex_%D0%BA%D0%BE%D0%BD%D1%82%D0%B5%D0%B9%D0%BD%D0%B5%D1%80%D0%B5)